Sources/RxViewController/UIViewController+Rx.swift

```diff
@@ -50,5 +50,39 @@ public extension Reactive where Base: UIViewController {
     let source = self.methodInvoked(#selector(Base.didReceiveMemoryWarning)).map { _ in }
     return ControlEvent(events: source)
   }
+
+    /// Rx observable, triggered when the ViewController has appeared for the first time
+    public var firstTimeViewDidAppear: Single<Void> {
```

Comment:
I'm not sure `firstTimeViewDidAppear` is more useful than just using `viewDidAppear.take(1)` 🤔